src/features/scheduling/FreeplayScheduler.ts

```
@@ -0,0 +1,44 @@
+  /* The current freeplay schedule */
+  schedule: Freeplay = { posts: {}, buddies: {} };
+
+  /* The session attendees that still need to be assigned */
+  campers: CamperAttendeeID[] = [];
+  staff: StaffAttendeeID[] = [];
+  admins: AdminAttendeeID[] = [];
+
+  /* The previous freeplay buddies from earlier in this session */
+  previousFreeplayBuddies: Freeplay[] = [];
+
+  constructor() { return this; }
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)